js/src/lib/components/InferenceWidget/shared/WidgetWrapper/WidgetWrapper.svelte

```diff
@@ -1,6 +1,6 @@
-<script lang="ts">
+<script lang="ts" generics="TWidgetExample extends WidgetExample">
```

Comment:
in the file [above](https://github.com/huggingface/hub-docs/pull/979/files#diff-db954a98b69d916d21056d16147e7c98bd981ec2174173d31d901f220991bd3e), we use `type TWidgetExample = $$Generic<WidgetExample>;`. Which style do we prefer?

Reply:
I guess we should go for `ttype TWidgetExample = $$Generic<WidgetExample>;` because there's something wrong with the tooling (prettier or eslint remove the type import :/ )

Reply:
care to push this change directly to the PR?

Reply:
e5cdad53b2867f80c849722dcf57742c8ad2981a